src/ControllerMethod.php

```diff
@@ -82,7 +82,7 @@ public static function parse(string $context, array $definitions, ClassMethod $m
 							} else {
 								$responseDescriptions[$statusCode] = $docNode->value->description;
 							}
-							$responses[] = new ControllerMethodResponse($docNode->value->type, $statusCode, "text/plain", new OpenApiType(type: "string"), null);
+							$responses[] = new ControllerMethodResponse($docNode->value->type, $statusCode, "application/json", new OpenApiType(type: "array", maxLength: 0), null);
```

Comment:
This change should only be applied when a OCSException was thrown: https://github.com/nextcloud/server/blob/ff2100a4e515988d3ba44c7cfe734d19a6b4b377/lib/private/AppFramework/Middleware/OCSMiddleware.php#L78
In all other cases the error still remains a text/plain string.

Reply:
Well the problem is it's `instanceof` in PHP, that means inheritance works, but it does not work here as we don't load any code...

Reply:
right, but should document this somewhere at some point...

Reply:
Yeah, but that is general OCS documentation and not specific to OpenAPI